twake/frontend/src/app/scenes/Client/MainView/Tabs/TabsTemplateEditor.tsx

```diff
@@ -32,21 +33,43 @@ export default (props: PropsType): JSX.Element => {
 
   return (
     <ObjectModal
-      title="Create a new tab"
+      title={
+        props.tab?.data.id
+          ? Languages.t('scenes.client.mainview.tabs.tabstemplateeditor.title_tab_edition', [
+              props.tab?.data.name,
+            ])
+          : Languages.t('scenes.client.mainview.tabs.tabstemplateeditor.title_tab_creation')
+      }
       closable
       footer={
         <Button
           type="primary"
           onClick={() => {
+            let editedTab: TabResource | undefined = props.tab;
+
             ModalManager.closeAll();
-            return props.onChangeTabs({
-              name: tabName,
-              application_id: appId,
-              configuration: {},
-            });
+
+            if (editedTab?.data.id) {
+              editedTab.data = {
+                ...editedTab.data,
+                name: tabName,
+                application_id: appId,
+                configuration: {},
+              };
+            } else {
+              editedTab = new TabResource({
+                name: tabName,
+                order: 'pos_' + new Date().getTime(),
+                owner: props.currentUserId,
+                application_id: appId,
+                configuration: {},
+              });
+            }
+
+            return props.onChangeTabs(editedTab);
           }}
         >
-          Create
+          {Languages.t(props.tab?.data.id ? 'general.edit' : 'general.create')}
         </Button>
```

Comment:
We must not be able to change the app in edition (so hide the select in edition)

Reply:
If you could also set the name input and select input sizes to large:  <Select size={'large'}  <Input size={'large'}

Reply:
Done :+1: